extensions/wikia/CommunityPage/models/CommunityPageSpecialHelpModel.class.php

```diff
@@ -4,12 +4,14 @@ class CommunityPageSpecialHelpModel {
 	public function getData() {
 		return [
 			'title' => wfMessage( 'communitypage-help-module-title' )->plain(),
+			'titleIcon' => file_get_contents( 'extensions/wikia/CommunityPage/images/help_icon.svg' ),
```

Comment:
I treat help icons as parts of a common visual language, so as such I recommend to put this icon onto the design system unless it's not compliant